standup/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="/images/index.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="/standup/style.css" />
+    <title>User Standup | Real Dev Squad</title>
+  </head>
+
+  <body>
+    <nav class="navbar">
+      <p class="user-standup-nav">User Standup</p>
```

Comment:
Nit: It is not paragraph i.e. it is heading of website
use heading tag h1,h2,h3,h4,h5,h6